assets/js/base/context/cart-checkout/billing/index.js

```diff
@@ -29,15 +29,12 @@ export const useBillingDataContext = () => {
 export const BillingDataProvider = ( { children } ) => {
 	const [ billingData, setBillingDataState ] = useState( DEFAULT_STATE );
 
-	const setBillingData = useCallback(
-		( newData ) => {
-			setBillingDataState( ( prevState ) => ( {
-				...prevState,
-				...newData,
-			} ) );
-		},
-		[ setBillingDataState ]
-	);
+	const setBillingData = useCallback( ( newData ) => {
+		setBillingDataState( ( prevState ) => ( {
+			...prevState,
+			...newData,
+		} ) );
+	}, [] );
```

Comment:
This dependency wasn't needed; useState provides a safe function. https://reactjs.org/docs/hooks-reference.html#usestate

Reply:
Correct 👍 (and linting will not warn for it). But it also does no harm to include it (which is useful for just having a habit of adding dependencies). I'm not opinionated on this though so if folks remove or add the dependency I'm okay with either.